templates/archive-meetings.php

```diff
@@ -542,7 +542,11 @@ function tsml_body_class($classes)
 									<?php
 break;
 
-            case 'distance': ?>
+            case 'distance':
```

Comment:
this seems like it could be for a separate issue. also don't think we should not render the column at all, that could throw off the table's layout

Reply:
Good point.  I saw the display: none and it silence the xdebug notices on my local dev.

Reply:
Ah, might be quicker to just say echo @$meeting['distance'] below to silence the debug notice